src/pages/document.ejs

```diff
@@ -6,7 +6,10 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Pbench Dashboard</title>
-  <link rel="icon" href="/favicon.ico" type="image/x-icon">
+  <link rel="icon" href="<%= context.config.publicPath +'favicon.ico'%>" type="image/x-icon">
+  
+  <!-- runtime endpoints config -->
+  <script src="<%= context.config.publicPath +'endpoints.js'%>"></script>
```

Comment:
I haven't seen that `<%=` syntax before. What does that actually mean? I tried looking it up, but what I found wasn't really helpful. Is it just a Javascript environment to allow the path concatenation operator?

Reply:
This has been adapted from a framework called [ejs](https://github.com/mde/ejs). It allows for dynamic HTML templating and has some pretty cool features!